specification/network/resource-manager/Microsoft.Network/stable/2021-03-01/bastionHost.json

```diff
@@ -180,6 +180,62 @@
         "x-ms-long-running-operation-options": {
           "final-state-via": "azure-async-operation"
         }
+      },
+      "patch": {
+        "tags": [
+          "BastionHosts"
+        ],
+        "operationId": "BastionHosts_UpdateTags",
+        "description": "Updates Tags for BastionHost resource",
+        "parameters": [
+          {
+            "$ref": "./network.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/BastionHostName"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "./network.json#/definitions/TagsObject"
```

Comment:
Does PATCH only support tags? Are all other properties immutable?

Reply:
At the moment yes, only tags are supported.

Reply:
For other parameters, users can use PUT operation @j5lim

Reply:
PUT is for creation of replacement of a resource. I'll approve this for now but please consider supporting partial update via PATCH (not just tags).

https://armwiki.azurewebsites.net/api_contracts/guidelines/rpc.html#rpc009-use-put-for-update-of-entire-resource-ie-replace-use-patch-only-for-partial-update-of-resource